### ISSUE TITLE
Can't see the "Releases" field in Jira tasks

### ISSUE THREAD

**Issue by yohanb** [microsoft/azure-pipelines-jira#35]
Hi,
I'm getting the same error reported in #15 .
I looked at the available issue fields in the configuration and it isn't there.
![image](https://user-images.githubusercontent.com/812993/82092870-09303000-96c8-11ea-9d9b-9f301d4f6601.png)

I also took the time to uninstall/reinstall the jira app and Azure DevOps service connection.
Can you please help me out?
Thanks.

**Comment by steenole** [comment]
This could very well be the same problem, that I have encountered. The problems is in JIRA.
Please read the comments on these two issues:

https://jira.atlassian.com/browse/JRACLOUD-73330

https://community.atlassian.com/t5/Jira-Software-questions/Releases-panel-in-old-issue-view/qaq-p/1198728?utm_campaign=&utm_content=post&utm_medium=email&utm_source=atlcomm#M78040

**Comment by yohanb** [comment]
yeah @steenole, seems to be the same problem!  Thanks for the info.  Hopefully this gets some attention because it basically makes the extension unusable.

**Comment by steenole** [comment]
@yohanb Please leave some comments and votes on the JIRA issues. The more of that, the more hope for some attention, I guess...

**Comment by yohanb** [comment]
@steenole done and done.  🤞

**Comment by yohanb** [comment]
Maybe someone in the dev team (@shashban) can escalate this with Atlassian?  It really kills the value for the integration with Azure DevOps.

**Comment by yohanb** [comment]
I guess the intern working on this integration is gone.... 😂
No love is given! :(

**Comment by shashban** [comment]
Hello folks ... 
As an owner for the integration, I must tender an apology here for not looking enough into the the issues here. 
We are working on a fix for the issue reported on the Atlassian marketplace (something went wrong) during the installation of the app.
Can I request one of you to report the second issue on https://developercommunity.visualstudio.com/content/problem/post.html?space=21
Thanks
